packages/vite/src/node/plugins/resolve.ts

```diff
@@ -555,10 +552,21 @@ function tryCleanFsResolve(
     if (dirStat?.isDirectory()) {
       if (possibleJsToTs) {
         // try resolve .js, .mjs, .mts or .jsx import to typescript file
-        const tsSrcPaths = getPotentialTsSrcPaths(file)
-        for (const srcPath of tsSrcPaths) {
-          if ((res = tryResolveRealFile(srcPath, preserveSymlinks))) return res
-        }
+        const type = path.extname(file)
```

Comment:
I think it's clearer if this is named `fileExt`

Reply:
good idea, copied the name from the other function without thinking much